common/OpTestHost.py

```diff
@@ -87,6 +89,26 @@ def username(self):
     def password(self):
         return self.passwd
 
+    def get_new_ssh_connection(self):
+        time.sleep(1)
+        outsuffix = time.strftime("%Y%m%d%H%M%S")
+        logfile = os.path.join(self.results_dir,"%s.log" % outsuffix)
+        print "Log file: %s" % logfile
+        logcmd = "tee %s" % (logfile)
+        logcmd = logcmd + "| sed -u -e 's/\\r$//g'|cat -v"
+        print "logcmd: %s" % logcmd
+        logfile_proc = subprocess.Popen(logcmd,
+                                             stdin=subprocess.PIPE,
+                                             stderr=subprocess.PIPE,
+                                             stdout=subprocess.PIPE,
+                                             shell=True)
+        print repr(logfile_proc)
+        logfile = logfile_proc.stdin
+        ssh = OpTestSSH(self.ip, self.user, self.passwd,
```

Comment:
Be sure to set_system on the new OpTestSSH object, I am working on new patch which automates the login and prompt settings and we use the system member and the ssh object to help track state.

Reply:
sure, will set it. Thanks.